charts/ndb-operator/templates/certificate-ndb-operator-serving-cert.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+    app.kubernetes.io/component: certificate
+    app.kubernetes.io/created-by: ndb-operator
+    app.kubernetes.io/instance: serving-cert
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: certificate
+    app.kubernetes.io/part-of: ndb-operator
+  name: {{ include "ndb-operator.fullname" . }}-serving-cert
+  namespace: {{.Release.Namespace}}
+spec:
+  dnsNames:
+  - ndb-operator-webhook-service.ndb-operator-system.svc
+  - ndb-operator-webhook-service.ndb-operator-system.svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: ndb-operator-selfsigned-issuer
```

Comment:
@tuxtof @akshmish 
Should these also be parameterized/templated?
Thinking about the following scenario:

If a user deploys the operator using a different name, let's say 'test-operator'.. Will the dnsName and issuerRef still match or be resolved?

Reply:
if you allow to change the web hook service name , you need to align with cert name too

Reply:
I don't think we should provide the users with such knobs. There are a few reasons that I can think of: 

services like webhooks are internal ones, users do not directly invoke them
too much customization may create more problems in the future in terms of testing and manageability 
we are going to need to do change all those as a part of every release prep